lib/pymedphys/_mosaiq/delivery.py

```diff
@@ -246,7 +248,7 @@ def collimation_to_bipolar_mm(mlc_a, mlc_b, coll_y1, coll_y2):
     return mlc, jaw
 
 
-def delivery_data_sql(connection, field_id):
+def delivery_data_sql(connection, field_id, include_rowvers=False):
```

Comment:
The only place in the codebase where this is utilised is below within `fetch_and_verify_mosaiq_sql`. So you are free to adjust the API instead to just always `include_rowvers` in the return :slightly_smiling_face:.

Given it's only used internally, and only used within this module, it might as well also be dropped to be a private function.